### ISSUE TITLE
[ds-tck] resource OSGI-INF/vendor.properties is missing

### ISSUE THREAD

**Issue by laeubi** [osgi/osgi#635]
I'm currently try to enable the DS-TCK for Eclipse PDE.

The testcase `Properties` declares a component like this:

https://github.com/osgi/osgi/blob/8ab4f46152fe43176b2a157f32e205a94487a481/org.osgi.impl.bundle.component.annotations/src/org/osgi/impl/bundle/component/annotations/Properties.java#L28-L31

but `OSGI-INF/vendor.properties` is not part of 
https://github.com/osgi/osgi/tree/main/org.osgi.impl.bundle.component.annotations

also the maven deployed TCK do not contain such a file:
https://central.sonatype.com/artifact/org.osgi/org.osgi.test.cases.component.annotations/versions

even searching the whole git repository seem not contain any file named `vendor.properties` ...

**Comment by laeubi** [comment]
@bjhargrave as it seems you have initially added this can you give some insight about the `vendor.properties` here?

**Comment by bjhargrave** [comment]
It does not need to exist in the bundle. The component.annotations TCK is to test the DS annotations processing tool (e.g. bnd) to make sure it generates the proper xml documents. So the resource name does not need to represent an actual resource in the bundle.

https://github.com/osgi/osgi/blob/eb360277f633b7640c8c3752213775395b9edfa8/org.osgi.test.cases.component.annotations/src/org/osgi/test/cases/component/annotations/junit/DSAnnotationsTestCase.java#L1398

is where the TCK tests the xml.

**Comment by laeubi** [comment]
> It does not need to exist in the bundle.

The question for me is more is it _intentional_ that the file is missing (e.g. one wants to test that a missing file is still included) in wich case maybe it is better called `missing.properties` to make it more clear, or was it intended to be there and just forgotten to add to the testcase in wich case I probably would add at least an empty one.

**Comment by bjhargrave** [comment]
> it _intentional_ that the file is missing

Yes. The components in the bundle are never used at runtime. The bundle is only a holder of ds xml documents for the TCK to validate.
